.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
-node_modules/
-lib/
+node_modules
+flow-typed
```

Comment:
Yeah, that's one of my biggest gripes so far with Flow. Aside from that, setting it up is WAAAAAAAAAY much easier than TypeScript and their `tsconfig.json`.